### ISSUE TITLE
monitoring v1 does not work on windows 2004 and 20h2

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#31152]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**
- K8s version - 1.19.7-rancher2-1, deployed a windows cluster with 3 linux nodes, 3 windows worker nodes AMI: Windows_Server-20H2-English-Core-ContainersLatest-2021.01.13.
- Enable monitoring v1 on the cluster
- Exporter node windows cluster monitoring fails to come up
<img width="1245" alt="Screen Shot 2021-01-31 at 5 23 53 PM" src="https://user-images.githubusercontent.com/26032343/106990733-b43abb00-6729-11eb-95f5-2e0b8aaac24f.png">

**Expected Result:**
Monitoring v1 should work on windows 2004 and 20h2

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): **On 2.5-head - commit id: `2ae4cf117a` and master-head - commit id: `59ebdc`**
- Installation option (single install/HA): single

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): windows custom
- Kubernetes version (use `kubectl version`):

```
any
```

**Comment by aiyengar2** [comment]
We don't have a wmi_exporter-package v0.0.3 that supports 20H2. We need to bump the wmi_exporter-package to v0.0.5+ for 20H2 + 2004.

**Comment by SheilaghM** [comment]
This will also be resolved when we finish the Monitoring V1/V2 tickets in 2.5.7.  Does this need to be done for 2.5.6?

**Comment by aiyengar2** [comment]
@sowmyav27 I tested this out locally using Monitoring V1 `v0.2.1` in the `dev-v2.5` branch, which has `wmi_exporter-package` with the tag `v0.0.5`.

I used the following AMIs on AWS:
- Windows Server 20H2: `Windows_Server-20H2-English-Core-ContainersLatest-2021.01.13`
- Windows Server 2004: `Windows_Server-2004-English-Core-ContainersLatest-2021.01.13`

For 2004, no changes were required and Monitoring was deployed successfully since the new image has a 2004 tag.

For 20H2, we needed to bump `wmi_exporter-package` to `v0.0.7`, so the workaround is to add the following override as part of the answers provided in Monitoring:
```
exporter-node-windows.image.tag: v0.0.7
```

With this workaround, both 20H2 and 2004 should be successfully deployed.

We should test 2004 without the workaround and 2004 + 20H2 with the workaround for this issue.

#### Why test the workaround instead of just bumping the image version in the chart?

@luthermonson and I talked about this offline. The reason for not bumping the image tag just yet is because `v0.0.6` bumped the version of `wmi_exporter` that was packaged from `0.10.2` to `0.12.0`; this was released out-of-band for existing users, who are currently testing it as part of another workaround.

As a result, even though `v0.0.7`'s change is independent of that (e.g. the change is to just create a symlink that has `pwsh.exe` point to `powershell.exe`, which is necessary since we call pwsh as part of copying the wmi_exporter binary to the host path [here](https://github.com/rancher/system-charts/blob/655d14b14792c6f49d2eade3cd3129d8850278d2/charts/rancher-monitoring/v0.2.1/charts/exporter-node-windows/templates/daemonset.yaml#L27-L29)), we want to avoid bumping the overall tag version for `v0.0.7` before getting feedback from users who are currently testing `v0.0.6`.

cc: @deniseschannon @SheilaghM

**Comment by sowmyav27** [comment]
**On 2.5-head commit d: `3e96d7fb1c`**

- Deployed a windows 2004 cluster - VXLAn using k8s 1.19.7-rancher2-1 - 1 etcd, 1 control plane and 1 linux worker nodes, 3 windows worker nodes. 
- Cluster is deployed successfully.
- Enabled monitoring v1 - v0.2.1. `rancher/wmi_exporter-package:v0.0.5`
- Monitoring is enabled successfully.
- Metrics are seen for the windows worker nodes
- on the Prometheus page, go to Status → Targets  --> all targets on this page have the state to be up

**Comment by thehejik** [comment]
*Rancher v2.5-head 3e96d7f*

* Deployed windows cluster with kubernetes `v1.19.7-rancher2-1`, flannel with VXLAN - 1 etcd, 1 control plane and 1 linux worker nodes, 3 Windows_Server_20H2 worker nodes
* AWS region `us-east-2`
* Used windows AMI: `ami-005281e398ece7d23` - Windows_Server-20H2-English-Core-ContainersLatest-2021.01.13
* Enabled monitoring v1 - v0.2.1 with modified `exporter-node-windows-cluster-monitoring` **DaemonSet using *custom* image** `rancher/wmi_exporter-package:v0.0.7` **inc. the sidecar**
* Monitoring works and metrics are seen in Amber UI; grafana; all targets for prometheus are UP

The initial `wmi_exporter-package:v0.0.5` image is failing on following and the corresponding pods didn't start.
```
Error: failed to start container "exporter-node-binary-copy": Error response from daemon: container exporter-node-binary-copy encountered an error during hcsshim::System::CreateProcess: failure in a Windows system call: The system cannot find the file specified. (0x2) extra info: {"CommandLine":"pwsh -f c:/scripts/copy-binary.ps1","WorkingDirectory":"C:\\","Environment":{"ACCESS_GRAFANA_PORT":"tcp://10.43.6.8:80","ACCESS_GRAFANA_PORT_80_TCP":"tcp://10.43.6.8:80","ACCESS_GRAFANA_PORT_80_TCP_ADDR":"10.43.6.8","ACCESS_GRAFANA_PORT_80_TCP_PORT":"80","ACCESS_GRAFANA_PORT_80_TCP_PROTO":"tcp","ACCESS_GRAFANA_SERVICE_HOST":"10.43.6.8","ACCESS_GRAFANA_SERVICE_PORT":"80","ACCESS_GRAFANA_SERVICE_PORT_HTTP":"80","ACCESS_PROMETHEUS_PORT":"tcp://10.43.150.71:80","ACCESS_PROMETHEUS_PORT_80_TCP":"tcp://10.43.150.71:80","ACCESS_PROMETHEUS_PORT_80_TCP_ADDR":"10.43.150.71","ACCESS_PROMETHEUS_PORT_80_TCP_PORT":"80","ACCESS_PROMETHEUS_PORT_80_TCP_PROTO":"tcp","ACCESS_PROMETHEUS_SERVICE_HOST":"10.43.150.71","ACCESS_PROMETHEUS_SERVICE_PORT":"80","ACCESS_PROMETHEUS_SERVICE_PORT_NGINX_HTTP":"80","ARCH":"amd64","KUBERNETES_PORT":"tcp://10.43.0.1:443","KUBERNETES_PORT_443_TCP":"tcp://10.43.0.1:443","KUBERNETES_PORT_443_TCP_ADDR":"10.43.0.1","KUBERNETES_PORT_443_TCP_PORT":"443","KUBERNETES_PORT_443_TCP_PROTO":"tcp","KUBERNETES_SERVICE_HOST":"10.43.0.1","KUBERNETES_SERVICE_PORT":"443","KUBERNETES_SERVICE_PORT_HTTPS":"443","WMI_EXPORTER_ARCHIVER_VERSION":"0.10.2"},"CreateStdInPipe":true,"CreateStdOutPipe":true,"CreateStdErrPipe":true,"ConsoleSize":[0,0]}
```

**Comment by thehejik** [comment]
Going to test Windows 2004 with v0.0.7 exporter as requested above.

**Comment by thehejik** [comment]
Rancher v2.5-head 3e96d7f

The `exporter-node-windows-cluster-monitoring` DaemonSet using not-yet-released image `rancher/wmi_exporter-package:v0.0.7` is also working ok with `Windows_Server-2004-English-Core-ContainersLatest-2021.01.13` `ami-0ddedac8d6144fb58` (us-east-2)

* Deployed windows cluster with kubernetes v1.19.7-rancher2-1, flannel with VXLAN - 1 etcd, 1 control plane and 1 linux worker nodes, 3 Windows_Server-2004 worker nodes
* prometheus targets UP
* graphs in Amber UI 
* all resources Ready

**Comment by thehejik** [comment]
Opened follow up issue for Rancher v2.5.7 https://github.com/rancher/rancher/issues/31337
